packages/navstar/extra/dcos-navstar.service

```diff
@@ -26,4 +26,5 @@ ExecStartPre=/usr/bin/mkdir -p /var/lib/dcos/navstar/minuteman/dets
 ExecStartPre=/usr/bin/env modprobe dummy
 ExecStartPre=-/usr/bin/env ip link add minuteman type dummy
 ExecStartPre=/usr/bin/env ip link set minuteman up
+ExecStartPre=$DISABLE_BRIDGE_NETFILTER
```

Comment:
@aeyakovenko this might be the moment where we move all of this to a script. Maybe you're doing that already?